constants/currency.ts

```diff
@@ -44,13 +44,14 @@ export const CRYPTO_CURRENCY = [
 
 export const CRYPTO_CURRENCY_MAP = keyBy(CRYPTO_CURRENCY);
 
-export const FIAT_SYNTHS: Set<CurrencyKey> = new Set([
+export const FIAT_SYNTHS: Set<CurrencyKey | 'EUR'> = new Set([
 	Synths.sEUR,
 	Synths.sJPY,
 	Synths.sUSD,
 	Synths.sAUD,
 	Synths.sGBP,
 	Synths.sCHF,
+	'EUR',
```

Comment:
not sure if we want to do something like this here to avoid additional fiats not registering as the release:

```
export type FUTURES_FIAT = 'EUR' | 'JPY' | 'USD' | 'AUD' | 'GBP' | 'CHF';
export const FIAT_SYNTHS: Set<CurrencyKey | FUTURES_FIAT> = new Set([
	Synths.sEUR,
	Synths.sJPY,
	Synths.sUSD,
	Synths.sAUD,
	Synths.sGBP,
	Synths.sCHF,
	'EUR',
	'JPY',
	'USD',
	'AUD',
	'GBP',
	'CHF',
]);
```

Reply:
I think this is a good idea. Also something for us to note when we start writing our own queries and types from scratch.